gateway/src/apicast/policy/rate_limit/apicast-policy.json

```diff
@@ -196,7 +197,8 @@
             "window": {
               "type": "integer",
               "description": "The time window in seconds before the request count is reset",
-              "exclusiveMinimum": 0
+              "exclusiveMinimum": 0,
+              "default": 1
```

Comment:
Why is the default needed? It's required and there's a minimum value.

Also, this made me think about the limit in `count` (I know it's not part of the PR), but it's also `exclusiveMinimum: 0`. Not sure if that's correct. Should we allow 0 to reject always?

Reply:
Is > 0 
https://json-schema.org/understanding-json-schema/reference/numeric.html#range